Dockerfile

```diff
@@ -0,0 +1,87 @@
+FROM ubuntu:bionic
+WORKDIR /root
+
+# Install bitcoind
+RUN apt-get update \
+  && apt-get install -y software-properties-common \
+  && add-apt-repository ppa:bitcoin/bitcoin \
+  && apt-get update \
+  && apt-get install -y \
+    build-essential \
+    libtool \
+    autotools-dev \
+    automake \
+    pkg-config \
+    libssl-dev \
+    libevent-dev \
+    bsdmainutils \
+    python3 \
+    libboost-all-dev \
+    miniupnpc \
+    libzmq3-dev \
+    libdb4.8-dev \
+    libdb4.8++-dev \
+    git
+RUN git clone https://github.com/bitcoin/bitcoin.git \
```

Comment:
This can be done much simpler by just fetching a tarball and unpacking it in the correct place. There is no added value in compiling it in docker imho.

Reply:
If my understanding is correct, current lightning-integration doesn't work with officially released bitcoind binary.
#17 makes lightning-integration only work with bitcoind HEAD.
so I fetch it, then compile it.